modules/serverless-functions-secrets-configmaps-manually.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-manually_{context}"]
+= Adding function access to secrets and config maps manually
+
+You can manually add configuration for accessing secrets and config maps to your function.
+
+.Procedure
+
+. Open the `func.yaml` file for your function.
+
+. For each secret you want to mount as a volume, add the following YAML to the `volumes` section:
++
+[source,yaml]
+----
+name: test
+namespace: ""
+runtime: go
+...
+volumes:
+- secret: mysecret
```

Comment:
I'd add an explanation to the fields as well, that under secret users should put the name of the secret and under path should be the path where the secret should be mounted.